plugins/chat/assets/javascripts/discourse/components/user-menu/chat-notifications-list-empty-state.gjs

```diff
@@ -0,0 +1,21 @@
+import getUrl from "discourse/helpers/get-url";
+import htmlSafe from "discourse/helpers/html-safe";
+import { i18n } from "discourse-i18n";
+
+<template>
```

Comment:
this one didn't get named?

Reply:
Fixed manually. The codemod probably missed it because it [had this unusual JS file](https://github.com/discourse/discourse/blob/b85654e42547b5a9b5f760bbe92fe300149edec6/plugins/chat/assets/javascripts/discourse/components/user-menu/chat-notifications-list-empty-state.js).